src/style.css

```diff
@@ -0,0 +1,154 @@
+/* #region Normalize */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
```

Comment:
body does not have padding by default so this is redundant

Reply:
understood. thanks.